multimodal/health_multimodal/image/model/model.py

```diff
@@ -102,6 +100,11 @@ def get_patchwise_projected_embeddings(self, input_img: torch.Tensor, normalize:
 
 
 class ImageEncoder(nn.Module):
+    """
+    Image encoder trunk module for the ImageModel class.
+    :param img_model_type: Type of image model to use {"resnet18", "resnet50"}
```

Comment:
This needs a new line before. These are actually not just for style: the docs build will break otherwise.